tests/core/test_actions.py

```diff
@@ -255,10 +270,14 @@ async def test_remote_action_utterances_with_none_values(
 
     response = {
         "events": [
-            {"event": "form", "name": "restaurant_form", "timestamp": None},
+            {
```

Comment:
why did we need to adapt these?

Reply:
Because I had enforced type `number` for `timestamp` - but now that it's type Any, I can revert this change.

Reply:
Should be `Optional[float]` - let's revert though 👍🏻